src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugRpcModule.cs

```diff
@@ -272,5 +273,10 @@ public ResultWrapper<bool> debug_resetHead(Keccak blockHash)
             _debugBridge.UpdateHeadBlock(blockHash);
             return ResultWrapper<bool>.Success(true);
         }
+
+        public Task<ResultWrapper<SyncReportSymmary>> debug_getSyncStage()
+        {
+            return ResultWrapper<SyncReportSymmary>.Success(ReportSink.Snapshot);
```

Comment:
Why not make it based on `ISyncModeSelector` instead of this `ReportSink` static?

Reply:
I added an ISyncModeSelector to IDebugBridge, so I get the data from it, it was a static class before because it used to have aggregate fields for Time of sync and duration .. etc, but they were removed last minute.